.buildkite/pipeline.yml

```diff
@@ -1,4 +1,17 @@
 steps:
+  # Push up new images to DockerHub when they get to master
```

Comment:
Would be nice to say why we push the images: for faster during development/CI where nothing changes in the build system.

Reply:
Maybe also where the credentials are stored: exposed as env vars in the buildkite agent hook on each agent.